### ISSUE TITLE
run "carthage update" in command line

### ISSUE THREAD

**Issue by seairy** [ishkawa/APIKit#61]
Xcode 7 beta 5 caused APIKit/Request.swift to fail to build:

```
*** Fetching APIKit
*** Fetching Result
*** Checking out APIKit at "1.0.0-beta2"
*** Checking out Result at "0.6-beta.1"
*** xcodebuild output can be found in /var/folders/zs/nrb9j0ws1q79l86sbfstq3180000gn/T/carthage-xcodebuild.TMppnX.log
*** Building scheme "Result-Mac" in Result.xcodeproj
*** Building scheme "Result-watchOS" in Result.xcodeproj
*** Building scheme "Result-iOS" in Result.xcodeproj
*** Building scheme "APIKit-Mac" in APIKit.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64 /Users/user/Projects/Demo/Carthage/Checkouts/APIKit/APIKit/Request.swift
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
/Users/user/Projects/Demo/Carthage/Checkouts/APIKit/APIKit/Request.swift:92:51: error: 'stringByAppendingPathComponent' is unavailable: Use stringByAppendingPathComponent on NSString instead.
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64 /Users/user/Projects/Demo/Carthage/Checkouts/APIKit/APIKit/Request.swift
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(2 failures)
```

**Comment by ikesyo** [comment]
This is already addressed in #58, so you should specify `master` branch or 17af19306697b94ae61e73d6e4793010f13ad216 in your Cartfile at the moment.

**Comment by seairy** [comment]
thanks!
